drools-wb-screens/drools-wb-guided-rule-editor/drools-wb-guided-rule-editor-client/src/main/java/org/drools/workbench/screens/guided/rule/client/editor/GuidedRuleEditorPresenter.java

```diff
@@ -206,7 +206,7 @@ protected void save() {
                                                                              GuidedRuleEditorResources.CONSTANTS );
 
         if ( validator.isValid() ) {
-            saveOperationService.save( versionRecordManager.getPathToLatest(),
+            savePopUpPresenter.show( versionRecordManager.getPathToLatest(),
```

Comment:
@karreiro Could you please run the [KIE Code Style](https://github.com/droolsjbpm/droolsjbpm-build-bootstrap/tree/master/ide-configuration/intellij-configuration/code-style) over this class? It's a good habit to get into to format code before submitting.